echo_bot/mix.exs

```diff
@@ -33,6 +33,7 @@ defmodule EchoBot.Mixfile do
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
      {:gettext, "~> 0.11"},
-     {:cowboy, "~> 1.0"}]
+     {:cowboy, "~> 1.0"},
+     {:botbuilder, path: "../../elixir-bot-builder", app: false}]
```

Comment:
Shouldn't this be the github path? Since it will be public
This relative path is for development